### PR TITLE
fix: Update the command help and examples for deployment

### PR DESCRIPTION
**Pull request by asalkeld**

### CODE REVIEW

File: pkg/cmd/deployment/root.go [hunk 1]
```diff
@@ -71,9 +72,12 @@ var deploymentApplyCmd = &cobra.Command{
 }
 
 var deploymentDeleteCmd = &cobra.Command{
-	Use:   "delete [handlerGlob]",
+	Use:   "delete",
 	Short: "Delete an application deployment",
 	Long:  `Delete a Nitric application deployment.`,
+	Example: `nitric delete -n prod-aws
```
Comment (tjholm) [inline pkg/cmd/deployment/root.go]: Is there a alias/shortcut setup for this or should this be `nitric deployment delete`

File: pkg/cmd/deployment/root.go [hunk 2]
```diff
@@ -86,24 +90,21 @@ var deploymentDeleteCmd = &cobra.Command{
...
-	Args: cobra.MinimumNArgs(0),
+	Args: cobra.ExactArgs(0),
 }
 
 var deploymentListCmd = &cobra.Command{
-	Use:   "list [handlerGlob]",
+	Use:   "list",
 	Short: "list deployments for a stack",
 	Long:  `Lists Nitric application deployments for a stack.`,
+	Example: `nitric list
```
Comment (tjholm) [inline pkg/cmd/deployment/root.go:109]: same question for delete

**Review by tjholm — COMMENTED** [review]
Not sure if these are typos are actual shortcuts available in the CLI

File: pkg/cmd/deployment/root.go (hunk 1 above)
Comment (asalkeld) [reply pkg/cmd/deployment/root.go]: the deployment name defaults to "dep", so it's not required

**Pull request merged**